### PR TITLE
🛑✋[DO NOT MERGE] Add footer

### PR DESCRIPTION
**Pull request by hereisdx**
As a part of the inductions task, I have added a footer to most pages of both AUGSD and TD.
The footer uses mdl grids and is responsive.

The footer is included as a file 'footer.jade' in the Views folder.

Screenshots: 

<br>

Known Issues:
1. Several pages do not have enough content to show the footer at a sufficiently low position. In such cases, either I have changed background of the page ( ex AUGSD Home ) or forced increased height of main content ( AUGSD Dashboard-master ) .

### CODE REVIEW

**Comment by sohailrajdev97** [comment]
Several changes are needed in this PR:

1. Footer is only required at the homepage (before login)
1. Color of the footer needs to be changed. Current color does not suit well for our theme
1. Travis checks have failed
1. This PR adds a lot of files which are not relevant for this project. (`redis-stable` and other submodules)

**Comment by Kailash0311** [comment]
1.)Use `grunt fix` to prettify files and always run this before pushing.
2.) Move all the redis modules to your home directory.

File: app.js [hunk 1]
```diff
@@ -22,14 +22,14 @@ app.use(
   )
 );
 
-morgan.token("user", function(req) {
+morgan.token("user", function (req) {
```
Comment (Kailash0311) [inline app.js]: Travis checks have failed due to changes in this file. Revert this file to previous state.

File: views/TD/footer.jade [hunk 2]
```diff
@@ -0,0 +1,64 @@
+style.
+    .special-footer {
```
Comment (Kailash0311) [inline views/TD/footer.jade]: Why not just `.footer` ?

**Comment by Kailash0311** [comment]
Closing this as there is another PR opened for the same.

**Pull request closed without merging**